examples/http_sync_server.hpp

```diff
@@ -66,22 +69,44 @@ class http_sync_server
     }
 
     void
-    fail(error_code ec, std::string what)
+    log_args()
     {
-        std::cerr <<
-            what << ": " << ec.message() << std::endl;
     }
 
+    template<class Arg, class... Args>
     void
-    maybe_throw(error_code ec, std::string what)
+    log_args(Arg const& arg, Args const&... args)
```

Comment:
If you do choose to make `http_async_server::log_args()` private, consider making this `log_args()` private as well.